python/tchannel/tornado/connection.py

```diff
@@ -1,18 +1,25 @@
 from __future__ import absolute_import
 
 import functools
+import logging
+import os
+import socket
+import sys
+
+from tornado import gen
+from tornado import iostream
 
 from .. import frame
 from .. import messages
 from .. import exceptions
 from ..io import BytesIO
 from ..context import Context
+from ..messages import CallResponseMessage
 from ..messages.types import Types
 from ..messages.common import PROTOCOL_VERSION
 
-from tornado import gen
-from tornado.iostream import StreamClosedError
-from tchannel.messages import CallResponseMessage
+
+log = logging.getLogger('tchannel')
 
 
 class TornadoConnection(object):
```

Comment:
I don't think we need that if we have a good `Handler` interface. `TChannel`, being the `Connection` manager might have 2 buckets for inbound/outbound, but a `Connection` itself just needs to listen to the wire and take action when it sees messages. That would happen via these `Handlers`, so inbound/outbound would make more sense there. None of this API stuff is finalized though so whatever works right now ;)